ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/PaywallIcon.kt

```diff
@@ -0,0 +1,169 @@
+package com.revenuecat.purchases.ui.revenuecatui.composables
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.revenuecat.purchases.ui.revenuecatui.R
+import kotlin.random.Random
+
+@Composable
+internal fun PaywallIcon(
+    icon: PaywallIconName,
+    modifier: Modifier = Modifier,
+    tintColor: Color,
+) {
+    Icon(
+        modifier = Modifier
+            .aspectRatio(1.0f)
+            .fillMaxSize()
+            .then(modifier),
+        painter = painterResource(id = icon.drawable()),
+        contentDescription = null,
+        tint = tintColor,
+    )
+}
+
+/**
+ * An icon that can be displayed inside a RevenueCat paywall.
+ */
+enum class PaywallIconName(val value: String) {
+    ADD("add"),
+    ANDROID("android"),
+    APPLE("apple"),
+    ATTACH_MONEY("attach_money"),
+    ATTACHMENT("attachment"),
+    BAR_CHART("bar_chart"),
+    BOOKMARK("bookmark"),
+    BOOKMARK_NO_FILL("bookmark_no_fill"),
+    CALENDAR_TODAY("calendar_today"),
+    CHAT_BUBBLE("chat_bubble"),
+    CHECK_CIRCLE("check_circle"),
+    CLOSE("close"),
+    COLLAPSE("collapse"),
+    COMPARE("compare"),
+    DOWNLOAD("download"),
+    EDIT("edit"),
+    EMAIL("email"),
+    ERROR("error"),
+    EXPERIMENTS("experiments"),
+    EXTENSION("extension"),
+    FILE_COPY("file_copy"),
+    FILTER_LIST("filter_list"),
+    FOLDER("folder"),
+    GLOBE("globe"),
+    HELP("help"),
+    INSERT_DRIVE_FILE("insert_drive_file"),
+    LAUNCH("launch"),
+    LAYERS("layers"),
+    LINE_CHART("line_chart"),
+    LOCK("lock"),
+    NOTIFICATION("notification"),
+    PERSON("person"),
+    PHONE("phone"),
+    PLAY_CIRCLE("play_circle"),
+    REMOVE_RED_EYE("remove_red_eye"),
+    SEARCH("search"),
+    SHARE("share"),
+    SMARTPHONE("smartphone"),
+    STACKED_BAR("stacked_bar"),
+    STARS("stars"),
+    SUBTRACT("subtract"),
+    TICK("tick"),
+    TRANSFER("transfer"),
+    TWO_WAY_ARROWS("two_way_arrows"),
+    KEY("key"),
+    WARNING("warning"),
```

Comment:
I believe the strings always match the enum but uppercased right? If so, we could just remove the string value and in the `fromValue`, we can do `PaywallIconName.valueOf(value.uppercased())` I think. Or create a map from the lowercased strings to the enums as well, to avoid linear lookup

Reply:
Let me check 👍🏻

Reply:
Done!